### PR TITLE
[Core] Added changelog for 0.5.0

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: src/Core/CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,40 @@
 
 ## 0.5.0
 
+### Added
+
+- Internal `AsyncAws\Core\Response` to encapsulate the HTTP client.
```
Comment (jderusse) [inline src/Core/CHANGELOG.md:7]: should add or remove final dot for consistency

File: src/Core/CHANGELOG.md [hunk 2]
```diff
@@ -2,9 +2,40 @@
...
+- Internal `AsyncAws\Core\RequestContext`
+- DynamoDB support
+- `ResultMockFactory` was updated with `createFailing()` and support for pagination.
+- `AbstractApi::presign()`
+- `Result::wait()` for multiplexing downloads
+- Interface `AsyncAws\Core\Input`
+- `AsyncAws\Core\Stream\ResponseBodyResourceStream` and `AsyncAws\Core\Stream\ResponseBodyStream`
+- `AsyncAws\Core\Stream\RewindableStream`
```
Comment (jderusse) [inline src/Core/CHANGELOG.md:15]: it's internal right?

Comment (Nyholm) [reply src/Core/CHANGELOG.md:15]: Yes, it is

File: src/Core/CHANGELOG.md (hunk 1 above)
Comment (Nyholm) [reply src/Core/CHANGELOG.md]: Sorry. Thank you

File: src/Core/CHANGELOG.md (hunk 2 above)
Comment (Nyholm) [reply src/Core/CHANGELOG.md]: I updated the PR

**Comment by Nyholm** [comment]
PR is rebased

**Pull request merged**